### DISCUSSION TITLE
Using HoloISO to test SteamDeck compatibility

### DISCUSSION THREAD

**Discussion by TechnoFlawgic** [PhilipK/BoilR#150] [Ideas]
Since you don't have your Steam Deck yet, have you looked into virtualizing for testing features that are specific to that platform? I currently have a VM running [HoloISO](https://github.com/theVakhovskeIsTaken/holoiso) and thought it might be of use to you as well.

**Comment by PhilipK** [comment]
I have tried to run Steam OS as a vm to debug some things. Problem is that I cannot get it to enter "game mode" or to actually launch games. So it is useful to test some things, but others are not possible to test with it.

**Comment by TechnoFlawgic** [comment]
That's unfortunate, I had to tinker a bit with the resolution, but I've got it running currently with an AMD card in GPU passthrough.

**Reply by PhilipK** [reply]
Alright, if you got it working I will give it another try.
